projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-form-block-ai-composition/components/ai-assistant-toolbar-button/index.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * External dependencies
+ */
+import { aiAssistantIcon } from '@automattic/jetpack-ai-client';
+import { ToolbarButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+
+export default function AiAssistantToobarButton(): React.ReactElement {
+	const onToggle = () => console.log( 'toggle!' ); // eslint-disable-line no-console
+
+	return (
+		<ToolbarButton
+			showTooltip
+			onClick={ onToggle }
+			aria-haspopup="true"
+			aria-expanded={ true }
```

Comment:
This one is wrong. Should be good to make sure we fix it later.

Reply:
Do you have some issue related with improving this one? Important to add this there.

Reply:
yes, it's part of the migration process.https://github.com/Automattic/jetpack/issues/32160

Reply:
sorry, wrong line. I meant to this one: https://github.com/Automattic/jetpack/issues/32176